### PR TITLE
Add/migrate stat var info API used for NL/Disaster page data check

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: server/routes/api/variable_group.py [hunk 1]
```diff
@@ -36,7 +36,8 @@ def get_variable_group_info():
   """
   dcid = request.args.get("dcid")
   entities = request.args.getlist("entities")
-  result = dc.get_variable_group_info(dcid, entities)
+  resp = dc.get_variable_group_info([dcid], entities)
+  result = resp.get("data")[0].get("info")
```
Comment (beets) [inline server/routes/api/variable_group.py:40]: add checks or default return values for the dereferences?

**Review by beets — APPROVED** [review]
wow, large change! thanks for the refactor

File: server/routes/api/variable_group.py (hunk 1 above)
Comment (shifucun) [reply server/routes/api/variable_group.py]: Done

**Comment by shifucun** [comment]
Thanks!

**Pull request merged**